bigquery_etl/stripe/__init__.py

```diff
@@ -14,6 +14,22 @@
 import click
 import stripe
 
+# omit lists of resources that have a separate table
```

Comment:
The events for these resources are captured in those tables?

Reply:
yeah, more or less. the data is still captured, this is just reducing how much we repeat it.